modules/@apostrophecms/ui/ui/apos/stores/notification.js

```diff
@@ -31,9 +32,23 @@ export const useNotificationStore = defineStore('notification', () => {
       localize: options.localize
     };
 
-    // Send it to the server, which will send it back to us via polling
-    const { noteId } = await apos.http.post(apos.notification.action, { body: notif });
-    return noteId;
+    if (!options.clientOnly) {
+      // Send it to the server, which will send it back to us via polling
+      const { noteId } = await apos.http.post(apos.notification.action, { body: notif });
+      return noteId;
+    }
+
+    const clientNotif = {
+      _id: createId(),
+      updatedAt: new Date(),
```

Comment:
`updatedAt` is needed, used in notification polling to now what notifications to retrieve.